text.go

```diff
@@ -528,25 +528,18 @@ func (t *Text) MostCommonFontFace() FontFace {
 	return family.Face(size*ptPerMm, col, style, variant)
 }
 
-// ToPaths makes a path out of the text, with x,y the top-left point of the rectangle that fits the text (ie. y is not the text base)
-func (t *Text) ToPaths() ([]*Path, []color.RGBA) {
-	paths := []*Path{}
-	colors := []color.RGBA{}
+// RenderLetters renders the text converted to paths (calling r.RenderPath)
+// Don't forget to call RenderDecoration as well
+func (t *Text) RenderLetters(r Renderer, m Matrix) {
+	style := DefaultStyle
 	for _, line := range t.lines {
 		for _, span := range line.spans {
 			p, _, col := span.ToPath(span.width)
-			p = p.Translate(span.dx, line.y)
-			paths = append(paths, p)
-			colors = append(colors, col)
-		}
-		for _, deco := range line.decos {
-			p := deco.face.Decorate(deco.x1 - deco.x0)
-			p = p.Translate(deco.x0, line.y)
-			paths = append(paths, p)
-			colors = append(colors, deco.face.Color)
+			p = p.Translate(span.dx, line.y) // +deco.face.Voffset ?
+			style.FillColor = col
+			r.RenderPath(p, style, m)
 		}
 	}
-	return paths, colors
 }
 
 // RenderDecoration renders the text decorations using the RenderPath method of the Renderer.
```

Comment:
@tdewolff `RenderDecoration` already exists (and is a little bit different, see `+deco.face.Voffset`).

It would make sense to have `RenderDecoration` & `RenderLetters`, don't you think? (and try to reduce duplication)